src/styles/components/footer.scss

```diff
@@ -43,6 +43,10 @@ footer {
     }
   }
 
+  input {
```

Comment:
this is global. Maybe we can put it in with all the other global modifications?

Reply:
I'm not sure this is global. Most of the inputs will have a border, but the email input box on the footer does not.